argo-cd-apps/overlays/development/kustomization.yaml

```diff
@@ -55,4 +55,4 @@ patches:
     target:
       kind: ApplicationSet
       version: v1alpha1
-      name: image-controller
+      name: image-controller
```

Comment:
missing a new line at the end of the file.

Reply:
fixed